### ISSUE TITLE
LeptonX Lite theme - Dashboard stays active (colored) in left nav bar

### ISSUE THREAD

**Issue by lukagalovic** [abpframework/abp#14826]
### Description

I created dashboard component. With old theme it worked fine, but after migrating to LeptonX theme after I click it and then go to some other component it stays colored like it is active.

### Version

6.0.1

### User Interface

Angular
![DashboardBug](https://user-images.githubusercontent.com/84729436/203941684-2a7d7ba3-16e5-4bfa-9537-6515ff29ac29.png)

**Comment by fatihskilic** [comment]
I can't reproduce this issue. Can you share your router.providers and app.routing.ts ?

**Comment by lukagalovic** [comment]
app-routing.module.ts
`import { NgModule } from '@angular/core';`
`import { RouterModule, Routes } from '@angular/router';`

`const routes: Routes = [`
`{ path: '', redirectTo: 'main/dashboard', pathMatch: 'full' },`
`{ path: 'admin', loadChildren: () => import('./admin/admin.module').then(m => m.AdminModule) },`
`{ path: 'main', loadChildren: () => import('./main/main.module').then(m => m.MainModule) },`
`{ path: 'account', loadChildren: () => import('@abp/ng.account').then(m => m.AccountModule.forLazy()) },`
`{ path: 'identity', loadChildren: () => import('@abp/ng.identity').then(m => m.IdentityModule.forLazy()) },`
`{ path: 'tenant-management', loadChildren: () => import('@abp/ng.tenant-management').then(m => m.TenantManagementModule.forLazy()) },`
`{ path: 'setting-management', loadChildren: () => import('@abp/ng.setting-management').then(m => m.SettingManagementModule.forLazy()) },`
`{ path: '**', redirectTo: 'main/dashboard' }`
`];`

`@NgModule({
    imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
    exports: [RouterModule],
})`
`export class AppRoutingModule { }`




route.provider.ts
`import { RoutesService, eLayoutType } from '@abp/ng.core';`
`import { eThemeSharedRouteNames } from '@abp/ng.theme.shared';`
`import { APP_INITIALIZER } from '@angular/core';`

`export const APP_ROUTE_PROVIDER = [
    { provide: APP_INITIALIZER, useFactory: configureRoutes, deps: [RoutesService], multi: true },
];`

`function configureRoutes(routesService: RoutesService) {`
`return () => {`
`routesService.add([`
`{`
`path: '/',
                name: '::Menu:Dashboard',
                iconClass: 'fas fa-table',
                order: 1,
                requiredPolicy: 'Dashboard.Dashboard',
                layout: eLayoutType.application,`
`},`
`{
                path: './main/main',
                name: '::Menu:Main',
                iconClass: 'fas fa-list-ul',
                order: 2,
                layout: eLayoutType.application
            },`
`{
                path: './main/clients',
                name: '::Menu:Main:Clients',
                iconClass: 'fas fa-users',
                order: 3,
                requiredPolicy: 'Clients.Clients',
                layout: eLayoutType.application,
                parentName: "::Menu:Main"
            },`
`{
                path: './main/licences',
                name: '::Menu:Main:Licences',
                iconClass: 'fas fa-certificate',
                order: 4,
                requiredPolicy: 'Licences.Licences',
                layout: eLayoutType.application,
                parentName: "::Menu:Main"
            },`
`{
                path: './admin/applications',
                name: '::Menu:Administration:Applications',
                iconClass: 'fas fa-th',
                order: 5,
                requiredPolicy: 'Applications.Applications',
                layout: eLayoutType.application,
                parentName: eThemeSharedRouteNames.Administration,
            },`
`{
                path: './admin/licence-templates',
                name: '::Menu:Administration:LicenceTemplates',
                iconClass: 'fas fa-file-alt',
                order: 6,
                requiredPolicy: 'LicenceTemplates.LicenceTemplates',
                layout: eLayoutType.application,
                parentName: eThemeSharedRouteNames.Administration
            },`
`]);`
`};`
`}`

**Comment by mahmut-gundogdu** [comment]
I couldn't produce. I have tested. It work as expected. 
My test steps
- Create 6.0.2  app with angular.
- Add Main and Admin module.
- Add main, clients, license on Main, applications and license template on Admin.
- Add two module route with lazy load.
- Change configureRoutes fn with yours.
-  I tested with requiredPolicy then I tested without requiredPolicy. 

Nothing changed.  It works.


https://user-images.githubusercontent.com/2217899/212693971-420685a6-74c5-47ee-9a30-e987cad86f3f.mov



 I tested without policy guard. both are working.

https://user-images.githubusercontent.com/2217899/212694795-f90c5e20-439e-4418-97b6-a831cbc975d2.mov
